nomt/src/rw_pass_cell.rs

```diff
@@ -173,8 +325,14 @@ impl<T, Id> RwPassCell<T, Id> {
     /// Returns a handle to read the value. Requires showing a read pass.
     ///
     /// Panics if the provided pass belongs to a different [`RwPassDomain`] than the cell.
-    pub fn read<'a, 'pass>(&'a self, read_pass: &'pass ReadPass) -> ReadGuard<'a, 'pass, T> {
+    /// If this pass is constrained to a specific region, this panics if the item's ID is outside
+    /// of the region's exclusive access range.
```

Comment:
"exclusive *or not* access range"

Should the read pass be able to access exclusive and non-exclusive ids?